### PR TITLE
#48: Support changing playback speed

### PR DESCRIPTION
**Pull request by wongni**
- Add setPlaybackRate method to YoutubePlayerController class

### CODE REVIEW

**Comment by sarbagyastha** [comment]
@wongni Have you tested by changing playback rate for video ? I don't think changing playback rate works in the Youtube Iframe API.

**Comment by wongni** [comment]
Yes I did. It works fine at least in Android phones.

2019년 8월 11일 (일) 오후 7:25, Sarbagya Dhaubanjar <notifications@github.com>님이
작성:

File: lib/src/youtube_player.dart [hunk 1]
```diff
@@ -583,6 +583,14 @@ class YoutubePlayerController extends ValueNotifier<YoutubePlayerValue> {
   /// Hide YouTube Player annotations like title, share button and youtube logo.
   void forceHideAnnotation() => _evaluateJS('hideAnnotations()');
 
+  /// Set the playback rate.
+  /// Playback rates may include values like 0.25, 0.5, 1, 1.5, and 2.
+  void setPlaybackRate(double playbackRate) {
+    if (playbackRate > 0.0 && playbackRate <= 2.0) {
+      _evaluateJS('player.setPlaybackRate($playbackRate)');
```
Comment (sarbagyastha) [inline lib/src/youtube_player.dart]: change to `setPlaybackRate` instead of `player.setPlaybackRate`.

**Comment by sarbagyastha** [comment]
Already implementedd in **Version 4.1.0**

**Pull request closed without merging**